src/ui/components/Icon/styles.scss

```diff
@@ -32,3 +32,29 @@
   background: url('./user-dark.svg') center no-repeat;
   background-size: contain;
 }
+
+.Icon-browse {
+  background: url('./bullet-list.svg') 50% 50% no-repeat;
+  background-size: cover;
+  height: 30px;
+  margin-right: 1em;
```

Comment:
This should use `@include margin-end(1em)`. The margin styles for these icons should probably move into where they're being used, too. You can add `className="LandingPage-browse-icon"` to the `Icon` call and then move this to a rule in LandingPage.scss, same with the other two.